include/ObjectManager.php

```diff
@@ -4171,6 +4242,57 @@ public function modifyNagiosMainConfiguration($requestConf=NULL, $exportConfigur
 	}
 ########################################## DELETE
 
+/* LILAC - delete kinship link */
+public function deleteParentToHost($parentName, $childName, $exportConfiguration=FALSE){
+		
+	$error = "";
+	$success = "";
+	$code=0;
+	
+	try{
+
+		// Wants to add a parent 
+		$nhp = new NagiosHostPeer;
+		// Find host
+		$parentHost = $nhp->getByName($parentName);
+		if(!$parentHost) {
+			$code=1;
+			$error .= "Parent Host $parentName does not exists\n";
+		}
+
+		$childHost = $nhp->getByName($childName);
+		if(!$childHost) {
+			$code=1;
+			$error .= "Child Host $childName does not exists\n";
+		}
+
+		if($code==0){
+			$c = new Criteria();
+			$c->add(NagiosHostParentPeer::CHILD_HOST , $childHost->getId());
+			$c->add(NagiosHostParentPeer::PARENT_HOST, $parentHost->getId());
+			$parentRelationship = NagiosHostParentPeer::doSelectOne($c);
+			if($parentRelationship) {
+				$parentRelationship->delete();
+				$success  .= "That parent relationship been deleted.\n";
+			}else {
+				$code = 1;
+				$error.= "That parent relationship does not exist yet.\n";
+			}
+
+			if( $exportConfiguration == TRUE )
```

Comment:
No need to test a value of a boolean: if($exportConfiguration)
is enough isn't it?